src/components/VolumeIndicator/VolumeIndicator.vue

```diff
@@ -20,27 +20,53 @@
 -->
 
 <template>
-	<div v-show="audioPreviewAvailable"
-		class="volume-indicator-wrapper"
-		:style="{ 'height': wrapperHeight + 'px' }">
-		<span ref="volumeIndicator"
-			class="volume-indicator"
-			:class="{'volume-indicator--disabled': disabled}"
-			:style="{ 'height': currentVolumeIndicatorHeight + 'px' }" />
-	</div>
+	<span class="volume-indicator"
+		:style="{ height: size + 'px', width: size + 'px' }">
+		<span class="volume-indicator-primary"
+			:style="{
+				height: size - 1 - currentVolumeIndicatorHeight + 'px',
```

Comment:
I would go with 2 px border
```suggestion
				height: size - 2 - currentVolumeIndicatorHeight + 'px',
```